### DISCUSSION TITLE
源分享

### DISCUSSION THREAD

**Discussion by Hiram-Wong** [Hiram-Wong/ZyPlayer#48] [Announcements]
大家有好的源在此下留言，不定时更新，觉得烂的没人强求你用

- 在线cms整理
于 2024.7.17日起域名和服务器都即将到期，不再提供在线一键配置

- 本地包
  -  新上线源动力平台，希望使用者都能赞助一下，才能良性运营 https://tvshare.cn
     - 不要挂代理, 服务器较为便宜(没人赞助), 不支持太多线路(仅香港+美国), 一般来说`国内+美国`访问没问题
     - 该平台只是一个爱好者分享平台, `不保证数据可用性`
     - ip变动后需要重新授权, 觉得麻烦的自己整理本地包
  -  本地包 Discussions #180 
  -  需 `3.3.4` 及以后版本

- tvbox 说明
  - 仅支持部分数据
  - 不支持挂载pg
  - 不支持加密数据导入
  - 详细说明查看Discussions #60

**配置完之后点刷新刷新刷新或重启软件**
**如导入完软件打不开, 说明接口有脏数据, 请手动重新整理数据, 删除数据库文件, 杀所有进程重启软件**

**Comment by mlw221** [comment]
这个怎么用？测试一下

**Comment by elegantflamingo22** [comment]
I want join community to watch free videos, how can I get the source url ?

**Comment by 122778740** [comment]
怎么没看到源

**Comment by HuangEnDi** [comment]
老哥，这个源寄了，好像上午还正常，下午就寄了

**Reply by Hiram-Wong** [reply]
服务器被我玩挂了，服务器资源太少了

**Reply by HuangEnDi** [reply]
感觉还是兼容一下tvbox方便，那边大佬维护的源比较多，稳定性好一点

**Reply by Hiram-Wong** [reply]
和 tvbox 不是一个方向，能适配再适配了

**Reply by HuangEnDi** [reply]
感谢大佬，zy在pc上还是很好用的

**Comment by cooldcc** [comment]
大佬这个源失效了  8点多我还再用  9点开始就连不上了

**Reply by Hiram-Wong** [reply]
如上，服务器被我玩挂了，服务器没资源

**Reply by cooldcc** [reply]
2核2G 40G 3M带宽  这个aliyun当服务器够用吗？  可以的话我给你买个去

**Reply by Hiram-Wong** [reply]
老板大气，够了

**Reply by cooldcc** [reply]
操作系统要选择哪个？

**Reply by Hiram-Wong** [reply]
centos

**Reply by Hiram-Wong** [reply]
感谢，有效期多久

**Reply by cooldcc** [reply]
一年 刚买的

**Reply by Hiram-Wong** [reply]
需要帮忙开下端口22041,80,443
或者开一个子账号给我也行
这是国内机器吗，国内机器的话我得上备案域名

**Reply by cooldcc** [reply]
是阿里云   3个端口都开放了  子账号我找找怎么开

**Reply by Hiram-Wong** [reply]
你是不是没开资源授权，看不到机器

**Reply by cooldcc** [reply]
开了 你试试  
![Uploading {F3EF2D66-0D64-47fe-8F45-6BF5E9A51BC3}.png…]()

**Reply by Hiram-Wong** [reply]
可以了，将上面域名换成 zy.hiram.top换成zy.catni.cn

**Reply by Hiram-Wong** [reply]
再发下之前的子账号，服务器被我玩挂了

**Comment by Hiram-Wong** [comment]
@ddwinhzy 帮忙重启下服务器

**Comment by huiwin** [comment]
@Hiram-Wong 大佬你的坚果泄密了、wy-hai@qq.com这个账号，通过zy.catni.cn\config.json导入的

**Comment by Hiram-Wong** [comment]
> @Hiram-Wong 大佬你的坚果泄密了、[wy-hai@qq.com](mailto:wy-hai@qq.com)这个账号，通过zy.catni.cn\config.json导入的

感谢提醒，已经修改授权码

**Comment by estherloverxx** [comment]
无法导入,修改成catni.cn一样无法导入

**Reply by cethink** [reply]
可以呀，我刚更新软件，重新导入，一秒成功

**Comment by WFNDVS** [comment]
这数据源可以自己建吗？可以的话该怎么建啊

**Reply by WFNDVS** [reply]
有大佬可以教教怎么建视频源吗

**Reply by Hiram-Wong** [reply]
1.搭建 cms+采集数据
2.drpy
3.hipy

**Comment by karldaens** [comment]
大佬你好，3.3.1版本的怎么导入呢，我在影视配置里添加，一直显示“网络请求失败请手动刷新”

**Reply by Hiram-Wong** [reply]
哪个地址

**Reply by karldaens** [reply]
就是换成zy.catni.cn的网址，我在浏览器网页上能打开这个网址，但是软件里识别显示无数据

**Reply by karldaens** [reply]
搞定了，是我傻了没摸清楚怎么配置

**Reply by leichtkunst** [reply]
配置不上啊，怎么搞的，哥0.0

**Reply by leichtkunst** [reply]
![Uploading Snipaste_2024-01-19_21-10-52.png…]()

**Reply by karldaens** [reply]
设置-基础配置-数据管理

**Reply by lifeisdox** [reply]
> 设置-基础配置-数据管理

我把2.8.8的源，导出来，在3.3.7里，在配置-基础配置-其他-数据管理-配置导入-本地导入，选择文件后，点导入，也显示导入成功，但是还是看不到任何源，请教下怎么回事？哪一步操作不对？

**Reply by Hiram-Wong** [reply]
> > 设置-基础配置-数据管理
> 
> 我把2.8.8的源，导出来，在3.3.7里，在配置-基础配置-其他-数据管理-配置导入-本地导入，选择文件后，点导入，也显示导入成功，但是还是看不到任何源，请教下怎么回事？哪一步操作不对？

2.8 和 3.3.7 不能直接导入，请按照readme 说明改数据格式后导入

**Comment by zouzhishi** [comment]
支持，点赞

**Comment by Hiram-Wong** [comment]
@cooldcc 哥，我看你2月27号执行了一条变更密码的操作，密码是多少

**Reply by Hiram-Wong** [reply]
@cooldcc 哥，在提供下cool666@1602122733509848.onaliyun.com这个子账号密码，之前的密码登不上了

**Comment by leojhltd** [comment]
大佬们，源好像失效了

**Reply by Hiram-Wong** [reply]
文件删错了 晚点处理

**Reply by leojhltd** [reply]
谢谢大佬持续关注

**Reply by Hiram-Wong** [reply]
已恢复

**Reply by leojhltd** [reply]
谢谢大佬，亲测可用

**Comment by wwb521** [comment]
源失效了，无法使用，能不能发我json文件，自己部署

**Reply by Hiram-Wong** [reply]
文件删错了 晚点处理

**Reply by Hiram-Wong** [reply]
已恢复

**Comment by chx66** [comment]
能不能加个去广告规则，还有解析怎么加到采集站里啊

**Comment by queand** [comment]
弹幕那个接口是什么啊 大佬们 我想看看弹幕 不会用啊

**Reply by torvalds4zh** [reply]
同问，目前播放器选了西瓜播放器还是看不到弹幕

**Reply by Hiram-Wong** [reply]
先配置弹幕 api，然后不是所有影片都有弹幕，弹幕库决定，一般官解都有，其他切片不一定有

**Comment by shangfe1** [comment]
那些18加的源呢

**Reply by Hiram-Wong** [reply]
哥们是想让我踩缝纫机吗

**Reply by 1728430580** [reply]
点播的接口还是用不了呀 点播是空白的

**Reply by Hiram-Wong** [reply]
> 点播的接口还是用不了呀 点播是空白的

没配置对呗

**Comment by DuyunliangToon** [comment]
大佬们，这个源[https://zy.catni.cn/api/config_v3.json]怎么不能解析视频了啊，换别的也不能解析啊

**Reply by Hiram-Wong** [reply]
1.设置错了
2.解析接口挂了
3.bugs

**Comment by cooldcc** [comment]
1986111aA



------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2024年8月5日(星期一) 下午5:07
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [Hiram-Wong/ZyPlayer] 源分享 (Discussion #48)














 
@cooldcc ***@***.***这个子账号密码，之前的密码登不上了

**Comment by bzw1204** [comment]
![image](https://github.com/user-attachments/assets/33eee832-4b66-4a85-82bd-beb3d3741dbd)
所有的源导入进去后都是提示这个

**Reply by thisharp** [reply]
哎，只有用手机用影视仓了，这东西源太少，能和影视仓源兼容就好了，不用开着电脑在开手机了

**Reply by Hiram-Wong** [reply]
> 哎，只有用手机用影视仓了，这东西源太少，能和影视仓源兼容就好了，不用开着电脑在开手机了

支持道长的js，测试版也支持xbpq，已经可以覆盖大部分了，同时pc不能加载安卓依赖的地位jar所以不肯呢兼容

**Reply by zomChen** [reply]
> > 哎，只有用手机用影视仓了，这东西源太少，能和影视仓源兼容就好了，不用开着电脑在开手机了
> 
> 支持道长的js，测试版也支持xbpq，已经可以覆盖大部分了，同时pc不能加载安卓依赖的地位jar所以不肯呢兼容

哥，我的macbook也提示这个，有没有源可以用呀？我不太懂

**Reply by Hiram-Wong** [reply]
> > > 哎，只有用手机用影视仓了，这东西源太少，能和影视仓源兼容就好了，不用开着电脑在开手机了
> > 
> > 
> > 支持道长的js，测试版也支持xbpq，已经可以覆盖大部分了，同时pc不能加载安卓依赖的地位jar所以不肯呢兼容
> 
> 哥，我的macbook也提示这个，有没有源可以用呀？我不太懂

源失效了，或者不兼容导致

**Comment by 2024Wangzhahz** [comment]
昨晚已经赞助了，知道楼主辛苦了，但是我自己是小白，确实不知道怎么配置，还望大佬指点

**Reply by 2024Wangzhahz** [reply]
求博主支持

**Reply by Hiram-Wong** [reply]
没懂，要配置啥

**Comment by 2024Wangzhahz** [comment]
就是这个影视配置






---原始邮件---
发件人: ***@***.***&gt;
发送时间: 2024年11月9日(周六) 中午12:40
收件人: ***@***.***&gt;;
抄送: ***@***.******@***.***&gt;;
主题: Re: [Hiram-Wong/ZyPlayer] 源分享 (Discussion #48)




 
没懂，要配置啥

**Reply by Hiram-Wong** [reply]
直接导入不就好了吗

**Comment by 2024Wangzhahz** [comment]
用下面的写源工具导入配置文件吗？
配置文件在哪里下载啊？



---原始邮件---
发件人: ***@***.***&gt;
发送时间: 2024年11月9日(周六) 中午1:28
收件人: ***@***.***&gt;;
抄送: ***@***.******@***.***&gt;;
主题: Re: [Hiram-Wong/ZyPlayer] 源分享 (Discussion #48)




 
直接导入不就好了吗

**Reply by Hiram-Wong** [reply]
自己找, 或者本地包 或者源动力 等等

**Comment by zz1204966173** [comment]
源动力网站验证通过后进不去，有什么办法解决吗？

**Reply by Hiram-Wong** [reply]
没太懂你的意思，源动力平台并没有加任何的限制，除了使用时进行 ip 授权

**Reply by zz1204966173** [reply]
![0af2f2cd-5cc5-42c5-9777-a9850e242e21](https://github.com/user-attachments/assets/3a6f663a-0656-4505-bd5b-ed9c4eaa5fbd)

**Reply by Hiram-Wong** [reply]
www.sourcepower.top

**Reply by zz1204966173** [reply]
就一直停留在这个界面了

**Reply by zz1204966173** [reply]
> [www.sourcepower.top](http://www.sourcepower.top)

好的，非常感谢

**Comment by Shiliu-daddy** [comment]
大佬，为什么我原动力导入之后，首页一直都是提示 哎呀出错了， 也搜索不出来任何数据呢？
可以看到列表中是有500多条数据的
通过快捷导入就提示：设置分类异常, 请前往设置检查源分类后尝试手动刷新
通过远端导入提示：哎呀出错了 

Win-3.3.8-x64
版本	Windows 11 专业版
版本号	23H2
安装日期	‎2023/‎3/‎6
操作系统版本	22631.4460
体验	Windows Feature Experience Pack 1000.22700.1047.0

**Reply by Hiram-Wong** [reply]
提供首页截图，和设置截图

**Reply by Shiliu-daddy** [reply]
![Snipaste_2024-11-20_18-08-02](https://github.com/user-attachments/assets/3f0e39ad-1b29-40b4-8b56-d59b15a68902)
![Snipaste_2024-11-20_18-08-24](https://github.com/user-attachments/assets/fed6868d-8d9a-46f1-89f7-5cee3f2ec828)
![Snipaste_2024-11-20_18-08-39](https://github.com/user-attachments/assets/0bbca726-9cbc-4f80-bc55-2f9fe98e80e7)
不知道能看倒么， 还是我自己方式搞错了

**Reply by Hiram-Wong** [reply]
请问源动力上 ip 授权了吗

**Reply by Shiliu-daddy** [reply]
是的 已授权，我和楼下这个一样， 方便远程todesk

**Reply by Hiram-Wong** [reply]
> 是的 已授权，我和楼下这个一样， 方便远程todesk

如果开了代理，请关闭

**Reply by Shiliu-daddy** [reply]
关掉了，还是一样效果呢，
我之前开了代是给浏览器用的，没有开全局，不然我联通打不开Github
关掉之后，尝试了重置数据+重新获取订阅并导入   还是一样提示 奇怪了

**Reply by Hiram-Wong** [reply]
> 关掉了，还是一样效果呢， 我之前开了代是给浏览器用的，没有开全局，不然我联通打不开Github 关掉之后，尝试了重置数据+重新获取订阅并导入 还是一样提示 奇怪了

上远程

**Reply by Leokanf** [reply]
也有这个问题

**Reply by Hiram-Wong** [reply]
> 也有这个问题

自行定位问题，源动力数据没问题

**Reply by Leokanf** [reply]
> > 也有这个问题
> 
> 自行定位问题，源动力数据没问题

请问ip授权怎么设置呢？

**Reply by Hiram-Wong** [reply]
> > > 也有这个问题
> > 
> > 
> > 自行定位问题，源动力数据没问题
> 
> 请问ip授权怎么设置呢？

源动力首页有说明

**Comment by leojhltd** [comment]
![image](https://github.com/user-attachments/assets/4c27f908-4d22-4367-894e-c25c3b2df153)
![image](https://github.com/user-attachments/assets/786a9542-6afe-4cab-867c-6e45823591e8)
已经订阅了，但是还是出错

**Reply by Hiram-Wong** [reply]
发个远程过来看下，todesk

**Reply by weills** [reply]
我也同上面的问题，订阅后，什么显示都没有

**Reply by Shiliu-daddy** [reply]
确认IP授权，订阅正常情况下，检查下网络 不要开任何代理相关
我的问题是之前搞的本地网易云音乐代理 导致网络失败了 卸载就行了   步骤流程操作对一般都没什么问题

**Reply by Shiliu-daddy** [reply]
> 我也同上面的问题，订阅后，什么显示都没有

确认IP授权，订阅正常情况下，检查下网络 不要开任何代理相关
我的问题是之前搞的本地网易云音乐代理 导致网络失败了 卸载就行了 步骤流程操作对一般都没什么问题

**Comment by cethink** [comment]
![image](https://github.com/user-attachments/assets/8a370dcc-dde6-442d-8c0e-5e33fcd24899)

我是这样导入的，有一些不能看，大部分可以

**Reply by cethink** [reply]
开了代理也没事

**Reply by Shiliu-daddy** [reply]
我也是这样导入的，显示数据有500多，但是首页啥都没有 搜索就提示 哎呀出错了

**Reply by cethink** [reply]
有些是会出错的，比如这个2千动漫就有内容
![image](https://github.com/user-attachments/assets/8442fa5a-1f5a-438f-a24b-1e33da15f41e)
我搜全站，就出来了
![image](https://github.com/user-attachments/assets/608df7a9-6e6d-4911-9367-56c08283a9cb)

**Reply by cethink** [reply]
> 我也是这样导入的，显示数据有500多，但是首页啥都没有 搜索就提示 哎呀出错了

![image](https://github.com/user-attachments/assets/230e2d2e-40f9-4169-8aae-7398bac7823f)
要设置默认，首页就有了

**Comment by weills** [comment]
搜索功能不能正常使用，只能在本站内搜索

**Reply by Hiram-Wong** [reply]
338已知 bug,下个版本修复

**Comment by Hanoch-J** [comment]
大佬，参考https://www.sourcepower.top// 首页下面建自己仓库的指导里提到的主要参数部分，现在的数据里的主要参数是都加密了吗？

**Reply by Hiram-Wong** [reply]
加密并不影响使用, drpy 后自动解密

**Reply by Hanoch-J** [reply]
好的，我尝试着本地拷了两个源，但是在zyfun 中数据导入时提示成功了，但是<影视>标签中不显示，追加的这两个源，麻烦帮我瞅一眼吧

![drpy_js_1](https://github.com/user-attachments/assets/43b5845d-c066-4ef1-95a7-69cb2c70a3f1)
![index](https://github.com/user-attachments/assets/00cafac3-c960-4dfe-ac13-971af9e00725)
分别对应，./drpy_js/[漫]樱漫.js 文件 和 ./index.json 文件

**Reply by Hiram-Wong** [reply]
type 不对，看 readme

**Comment by cethink** [comment]
https://www.sourcepower.top/ 打不开了

**Reply by Hiram-Wong** [reply]
正常打开

**Reply by cethink** [reply]
> 正常打开

可以了

**Comment by cethink** [comment]
覆盖500了，原先添加的也打不开了，提示：设置分类异常, 请前往设置检查源分类后尝试手动刷新

![image](https://github.com/user-attachments/assets/17645100-fb4e-4a64-8b9f-fbf2a3250a77)

**Reply by Hiram-Wong** [reply]
3310 会重复粘贴，自行确实

**Reply by cethink** [reply]
> 3310 会重复粘贴，自行确实

确实会重复粘贴，但只填一个也是这样的问题

**Reply by Hiram-Wong** [reply]
自行确认
1. 订阅链接浏览器打开是否有数据
2. 请勿开启 vpn 等具有网络转发等功能的软件会导致授权 ip 变动

**Reply by cethink** [reply]
> 自行确认
> 
> 1. 订阅链接浏览器打开是否有数据
> 2. 请勿开启 vpn 等具有网络转发等功能的软件会导致授权 ip 变动

我关了代理，重新订阅，订阅链接浏览器打开跳到 https://www.sourcepower.top/ 了，而且显示如下：

![image](https://github.com/user-attachments/assets/efe7f293-e250-4fb6-909b-f1b9f4b2224b)

**Reply by cethink** [reply]
订阅链接浏览器打开有数据，但还是报500错误

**Comment by cethink** [comment]
能不能加个批量检测

![image](https://github.com/user-attachments/assets/b3ad60a2-81af-40a4-bad0-5223f2e5e230)

**Comment by cethink** [comment]
在这里 https://www.juwanhezi.com/other/jsonlist 试了几个可用的源，分享一下

http://124.71.189.194/a.json
http://fmys.top/fmys.json
https://ztha.top/TVBox/thdjk.json
https://git.acwing.com/ocean2025/ocean/-/raw/main/api.json

**Comment by dajikute2** [comment]
在win10专业版，最新版3.4.0无法启动，可以看到三个进程，但是不能启动界面，是什么原因?

**Reply by cethink** [reply]
> 在win10专业版，最新版3.4.0无法启动，可以看到三个进程，但是不能启动界面，是什么原因?

卸载并删除所有相关目录文件，记得是所有，然后重装，点启动后窗口会闪个几十次，等着，最后启动成功就好了

**Reply by soyuzom** [reply]
确实这样，太魔幻了🤣

**Reply by xiaoxiaojingdo** [reply]
重新装回3.4.9了

**Comment by dajikute2** [comment]
多谢大佬回复解惑，晚上重装。发自我的手机-------- 原始邮件 --------发件人： OvO ***@***.***>日期： 2025年5月18日周日 上午10:50收件人： Hiram-Wong/ZyPlayer ***@***.***>抄送： dajikute2 ***@***.***>, Comment ***@***.***>主    题： Re: [Hiram-Wong/ZyPlayer] 源分享 (Discussion #48)

在win10专业版，最新版3.4.0无法启动，可以看到三个进程，但是不能启动界面，是什么原因?

卸载并删除所有相关目录文件，记得是所有，然后重装，点启动后窗口会闪个几十次，等着，最后启动成功就好了

**Comment by zz1204966173** [comment]
请问源动力有时会点登录没有反应，是什么原因呢

**Reply by Hiram-Wong** [reply]
由于各位同学不愿意分享，整体热度不高，源动力已于5月份下线

**Reply by zz1204966173** [reply]
> 由于各位同学不愿意分享，整体热度不高，源动力已于5月份下线
好的，谢谢
